iris-mpc-cpu/src/protocol/ops.rs

```diff
@@ -660,12 +659,11 @@ where
 {
     let start = Instant::now();
     let mut count = 0;
-    // * 31 for all rotations
-    // * 2 for code and mask distances
-    let mut additive_shares = Vec::with_capacity(31 * 2 * targets.len());
+    let mut additive_shares = Vec::with_capacity(2 * ROTATIONS * targets.len());
 
     for target in targets {
         for rotation in IrisRotation::all() {
+            count += 1;
```

Comment:
This looks unintended? This line is already there just below.

Reply:
good catch. was probably from rebasing to dev.